django/db/backends/sqlite3/operations.py

```diff
@@ -381,10 +381,6 @@ def combine_duration_expression(self, connector, sub_expressions):
             raise ValueError("Too many params for timedelta operations.")
         return "django_format_dtdelta(%s)" % ", ".join(fn_params)
 
-    def integer_field_range(self, internal_type):
-        # SQLite doesn't enforce any integer constraints
-        return (None, None)
```

Comment:
To keep this as backward compatible as we can we should use `[2**63, 2**63 - 1]` for all types.

Also, a release note in backward incompatible changes is missing.